src/metaopt/io/database/abstractdb.py

```diff
@@ -32,19 +32,19 @@ def close_connection(self):
         pass
 
     @abstractmethod
-    def insert(self, experiment_name, trial_field, dictionary, field_filters):
+    def insert(self, collection_name, dictionary, field_filters):
         pass
 
     @abstractmethod
-    def update(self, experiment_name, trial_field, update_dictionary, field_filters):
+    def update(self, collection_name, update_dictionary, field_filters):
         pass
 
     @abstractmethod
-    def load(self, experiment_name, experiment_field, field_filters):
+    def load(self, collection_name, field_filters):
```

Comment:
I think `(self, collection_name, query, projection)` or `(self, collection_name, query, selection)` would be more clear. `projection` is the name used by mongodb for what is called a _selection_ in SQL DBs. I don't have a clear preference. Sticking to mongodb's nomenclature could make the interface more coherent but at the same time I always found `projection` to be less obivous than _selection_.